macOS/DuckDuckGo/NavigationBar/View/AddressBarButtonsViewController.swift

```diff
@@ -516,8 +516,11 @@ final class AddressBarButtonsViewController: NSViewController {
         button.backgroundColor = .buttonMouseDown
         button.mouseOverColor = .buttonMouseDown
         (popover.contentViewController as? PermissionAuthorizationViewController)?.query = query
-        popover.show(relativeTo: button.bounds, of: button, preferredEdge: .maxY)
-        query.wasShownOnce = true
+
+        DispatchQueue.main.asyncAfter(deadline: .now() + NSAnimationContext.current.duration) {
```

Comment:
This change is based on @mallexxx 's recommendation [here](https://app.asana.com/1/137249556945/task/1210431866798492/comment/1210462202068011?focus=true).